src/Elastic.Transport/Requests/MetaData/VersionInfo.cs

```diff
@@ -74,6 +74,6 @@ public override string ToString()
 	{
 		var prefix = $"{Major}.{Minor}.{Patch}";
 
-		return IsPrerelease ? $"{prefix}p" : $"{prefix}";
+		return IsPrerelease ? $"{prefix}p" : prefix;
```

Comment:
I wonder if we should have a `ToMetadataHeaderValue()` and a `ToString()` that returns a semver compliant string?

That way `SemVersion.ToString()` actually roundtrips back to `SemVersion`?

Reply:
I added another commit that makes `VersionInfo` a wrapper around `SemVer`. This allows us to e.g. "passthrough" the `ToString()` to the wrapped `SemVersion`.